pdf/chunks/RankIndividual.inc.php

```
@@ -76,23 +76,14 @@
-					{
-						$pdf->Cell(12 - (strlen($v['tiebreak'])>0 && $k<=1 ? 6 : 0), 4, ($section['meta']['matchMode']==0 ? $v['score'] : $v['setScore']) . ($k<=1 && $v['tie']==1 && strlen($v['tiebreak'])==0 ? '*' : ''), ($k<=1 && strlen($v['tiebreak'])>0 ? 'LTB' : 1), 0, 'R', 0);
-						if(strlen($v['tiebreak'])>0 && $k<=1)
-							$pdf->Cell(6, 4,  "T.".str_replace('|',',',$v['tiebreak']), 'RTB', 0, 'R', 0);
-					}
+			$cntPhases=0;
+			foreach($item['finals'] as $k=>$v) {
+				$cntPhases++;
+				if($v['tie']==2) {
+					$pdf->Cell(12, 4, $PdfData->Bye, 1, 0, 'R', 0);
+				} else {
+					$pdf->Cell(5, 4, (($cntPhases<count($item['finals']) or floatval($v['avgTie'])==0) ? '' : ($v['avgTie'] ? number_format($v['avgTie'],3) : str_replace('|', ',', $v['tiebreak']))), 'LBT', 0, 'R', 0);
```

PHPStan (2.1.46)

[warning] Variable $pdf might not be defined.


(variable.undefined)